src/api/venue.ts

```diff
@@ -34,3 +44,71 @@ export const setVenueLiveStatus = async ({
     })
     .finally(onFinish);
 };
+
+// TODO: return an array of all venueId's we find on the full way up to the sovereignVenue
+export const fetchSovereignVenueId = async (
```

Comment:
> TODO: return an array of all venueId's we find on the full way up to the `sovereignVenue`

Reply:
fixed by @mike-lvov in https://github.com/sparkletown/sparkle/pull/1368/commits/1b0b472e3ceeab3f355dbbf4cdf1aae6612b170a